### ISSUE TITLE
Sync to FS

### ISSUE THREAD

**Issue by ClockGen** [msoultanidis/quillnote#9]
Would be nice to be able to sync notes to the local filesystem, for purposes of syncing them with things like syncthing or kde connect.

**Comment by meadows19** [comment]
I agree, but also I think it would be much better to just save the files in the local filesystem in the first place, and NOT have to 'sync' a copy of the files there manually.

**Comment by msoultanidis** [comment]
Syncing and importing/exporting to the filesystem as plain .MD files seem to be popular requests. I'll work on it. 
For now though, since they all are related feature requests let's move the discussion here.

**Comment by foxonfox** [comment]
Please do consider **.txt** files as well. many popular markdown editors treat .md and .txt files as same. When a note is created it could be created as a .txt/.md file in the file system itself this allows to sync it to PC easily. I always used .txt for my markdown notes and I would like to switch over to your note app (I love the design), the only thing stopping me is that there is no way to import my .txt files to your app.

**Comment by matm** [comment]
This would be a killer feature indeed. Like what Noteless does actually. This is very convenient with Syncthing.
Just storing the Markdown files in configurable location on the filesystem would be perfect.

BTW thank you very much for your work on this app @msoultanidis. Great UI/UX, open source, huge potential.

**Comment by whoibrar** [comment]
Yeah just discovered this app; loved the UI/UX and everything but when i searched searched through explorer to see the files that I've just created, I was surprised to see app doesn't auto store everything locally in simple .txt or .md files!

This would be a really cool feature, and is the only thing stopping me from switching over from iA Writer!

**Comment by b4skyx** [comment]
> I agree, but also I think it would be much better to just save the files in the local filesystem in the first place, and NOT have to 'sync' a copy of the files there manually.

Bump! I use syncthing to have my stuff locally across devices. The app is great, love the ui and its capabilities. 
Saving the files in the local filesystem in markdown would open more doors for backup and syncing.

**Comment by engelhro** [comment]
Came to here to say **congrats** 👏 on Quillnote as well – really slick and pretty app, feels a lot like Google keep (which I want to leave behind)! Thanks a lot for creating it! 👍

So I just want to chime in by supporting the requests above and ask for proper synchronization with other devices via the filesystem (using Syncthing in my case). 🙏

I'm used to edit notes and task lists *both* on the smartphone and my PC. And while I don't need a nice web UI on the desktop (like Google keep offers it in the browser, storing everything on their servers), I at least want to edit the respective cleartext files in a regular text / markdown editor in Windows.

As soon as that is possible, I could get rid of Google keep (except for the shared use of notes with other people, but for that I could still resort to sync via a  Nextcloud instance in the future).

Thanks a lot! 😃

**Comment by OmlineEditor** [comment]
to make it possible to select a folder for export, I want to choose where the folder will be, so it's more convenient

**Comment by desc4rtes** [comment]
This is a killer feature for me. Please give it more priority.

**Comment by Mrnofish** [comment]
Starred the project as Quillnotes seems pretty cool, however the lack of filesystem-based sync is a show-stopper.

**Comment by am01k01h3** [comment]
Well done with Quillnotes. Another vote for this feature.

Would love to be able to select directory to save notes by default (instead of a manual export/backup). This way I can sync the directory using syncthing across devices. Thanks!

**Comment by OmlineEditor** [comment]
> Any updates on this? I am unable to use this app as long as I cannot sync my notes via Syncthing. I currently use Markor for this, but would love to switch to Quillnote if it supports this feature!

the program was abandoned and is no longer being developed. there is no news now and there will be no news in the near future
